base/api/personApi.ts

```diff
@@ -0,0 +1,117 @@
+import { Page, expect, APIResponse } from '@playwright/test';
+import { befristungPflicht } from '../merkmale';
+import { generateCurrentDate, generateVorname, generateNachname } from '../utils/generateTestdata';
+import { FRONTEND_URL } from './baseApi';
+
+
+export interface UserInfo {
+  username: string;
+  password: string;
+  rolleId: string;
+  organisationId: string;
+  personId: string;
+}
+
+interface CreatedPersonResponse {
+  person: {
+    id: string,
+    referrer: string,
+    mandant: string,
+    name: {
+      familienname: string,
+      vorname: string
+    },
+    revision: string,
+    startpasswort: string,
+    lastModified: string
+  },
+  dBiamPersonenkontextResponses: [
+    {
+      personId: string,
+      organisationId: string,
+      rolleId: string
+    }
+  ]
+}
+
+export async function createPerson(
+  page: Page,
+  organisationId: string,
+  rolleId: string,
+  familienname?: string,
+  vorname?: string,
+  koPersNr?: string,
+  klasseId?: string,
+  merkmaleName?: string[]
+): Promise<UserInfo> {
+  interface PersonRequestData {
+    data: {
+      familienname: string;
+      vorname: string;
+      createPersonenkontexte: {
+        organisationId: string;
+        rolleId: string;
+      }[];
+    };
+    failOnStatusCode: boolean;
+    maxRetries: number;
+  }
```

Comment:
```suggestion
interface PersonRequestData {
  data: {
    familienname: string;
    vorname: string;
    createPersonenkontexte: {
      organisationId: string;
      rolleId: string;
    }[];
    personalnummer?: string;
    befristung?: string;
  };
  failOnStatusCode: boolean;
  maxRetries: number;
}
```
I would add  personalnummer and befristung as optional properties to make it known that they could be part of the request